src/index.js

```diff
@@ -6,10 +6,10 @@ const REGEX_INT = /^\d+$/;
 const REGEX_FLOAT = /^\d+(\.\d+)?$/;
 const REGEX_PHONE = /^(?=.{2,50}$)[/\-.()+\s\d–]*\d{2,}[/\-.()+\s\d–]*$/;
 
-const REGEX_LOWERCASE = /[a-z]/;
-const REGEX_UPPERCASE = /[A-Z]/;
-const REGEX_NUMBERS = /\d/;
-const REGEX_SPECIAL = /[^A-Za-z\d]/;
+const REGEX_LOWERCASE = /[a-z]+/;
+const REGEX_UPPERCASE = /[A-Z]+/;
+const REGEX_NUMBERS = /\d+/;
+const REGEX_SPECIAL = /[^A-Za-z\d]+/;
```

Comment:
This condition matches umlauts as well. I know you didn't change it in this PR but I suppose it's not intended

Reply:
Not sure where we use it. Wouldn't matter in passwords

Reply:
not sure if diacritics considered special, probably you're right. let's leave like it is until this is a problem